test/integration/spec/connect.js

```diff
@@ -990,6 +1013,73 @@ describe('connect', function() {
           assert.equal(loPriTrackPub.track.isSwitchedOff, true, 'low pri tracks were not switched off');
           assert.equal(hiPriTrackPub.track.isSwitchedOff, false, 'hi pri track were switched off');
         });
+
+        it('should switch off RemoteVideoTracks that are not published by the Dominant Speaker', async () => {
+          const [aliceTracks, bobTracks] = await Promise.all([1, 2].map(async () => [
```

Comment:
It feels it would look simpler w/o `Promise.all`

Reply:
@makarandp0 I think we should avoid redundant code as much as possible.